lib/diagnostic.rb

```diff
@@ -108,7 +108,7 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = '{@favorite_number = favorite_number,  @first_name = first_name}'
```

Comment:
Close but we don't need the @ symbol unless we are working with classes generally.

'person = {first_name: "Lauren", favorite_number: 7}'